test/unit/services/workers/test-generate-payout-payments.js

```diff
@@ -70,7 +70,7 @@ describe('services/workers/generate-payout-payments', function () {
       expect(getClaimsPendingPayment.calledOnce).to.be.true //eslint-disable-line
       expect(getTopUpsPendingPayment.calledOnce).to.be.true //eslint-disable-line
       expect(createPayoutFile.calledWith(claimsPendingPayment)).to.be.true //eslint-disable-line
-      expect(sftpSendPayoutPaymentFile.calledWith(`/tmp/${testPath}`, './test.csv')).to.be.true //eslint-disable-line
+      expect(sftpSendPayoutPaymentFile.calledWith(`/app/tmp/${testPath}`, './test.csv')).to.be.true //eslint-disable-line
```

Comment:
This is a bad test, well it's supposed to pull from config, but as there are numerous problems in these apps they can go down as tech debt for now.